go.mod

```diff
@@ -8,9 +8,8 @@ require (
 	github.com/operator-framework/api v0.4.0
```

Comment:
Need to bump this. Waiting on https://github.com/operator-framework/api/pull/94

Reply:
This can be updated to v0.5.2